src/modules/QtQuick/Item.js

```diff
@@ -41,7 +41,10 @@ function QMLItem(meta) {
     this.dataChanged.connect(this, function(newData) {
         for (var i in newData) {
             var child = newData[i];
-            if (child.hasOwnProperty("parent")) // Seems to be an Item. TODO: Use real inheritance and ask using instanceof.
+
```

Comment:
Yeah, but any reference to setting the Timer parent I wrote in these comments is now misleading as the Timer parent doesn't have to be set here and is handled on its own.